sentry-contrib-native-sys/build.rs

```diff
@@ -40,36 +40,53 @@ fn main() -> Result<()> {
         );
     }
 
+    let target_os = env::var("CARGO_CFG_TARGET_OS").expect("target os not specified");
+
     if !installed {
         build(&out_dir, &source, &install)?;
     }
 
-    println!("cargo:rustc-link-search={}", install.join("lib").display());
-    println!("cargo:rustc-link-lib=sentry");
-    println!("cargo:rustc-link-lib=mini_chromium");
-
-    let target_os = env::var_os("CARGO_CFG_TARGET_OS").unwrap();
-
-    if target_os == "windows" {
-        println!("cargo:rustc-link-lib=dbghelp");
-        println!("cargo:rustc-link-lib=shlwapi");
-        println!("cargo:rustc-link-lib=winhttp");
-    }
-
-    if target_os == "windows" || target_os == "macos" {
-        println!("cargo:rustc-link-lib=crashpad_client");
-        println!("cargo:rustc-link-lib=crashpad_util");
+    let lib_path = if target_os == "windows" {
+        install.join("lib")
+    } else {
+        install.join("lib64")
+    };
 
-        let handler = if target_os == "windows" {
-            "crashpad_handler.exe"
-        } else {
-            "crashpad_handler"
-        };
+    println!("cargo:rustc-link-search={}", lib_path.display());
+    println!("cargo:rustc-link-lib=sentry");
 
-        println!(
-            "cargo:HANDLER={}",
-            install.join("bin").join(handler).display()
-        );
+    match target_os.as_str() {
+        crashpad if crashpad == "windows" || crashpad == "macos" => {
+            println!("cargo:rustc-link-lib=crashpad_client");
+            println!("cargo:rustc-link-lib=crashpad_util");
+            println!("cargo:rustc-link-lib=mini_chromium");
+
+            let handler = if crashpad == "windows" {
+                println!("cargo:rustc-link-lib=dbghelp");
+                println!("cargo:rustc-link-lib=shlwapi");
+
+                if cfg!(not(feature = "custom-transport")) {
```

Comment:
Is there some advantage over using `#[cfg(...)]` here?

Reply:
This is easier to write as it's we don't have to worry about either branch not compiling.